deltachat-ios/Controller/MapViewController.swift

```diff
@@ -0,0 +1,33 @@
+import UIKit
+import WebKit
+import DcCore
+
+class MapViewController: WebxdcViewController {
+    private let isGlobalMap: Bool
+
+    init(dcContext: DcContext, chatId: Int) {
+        isGlobalMap = chatId == 0
+        var webxdcInstanceId = dcContext.initWebxdcIntegration(for: chatId)
+        if webxdcInstanceId == 0 {
+            if let mapsXdc = Bundle.main.url(forResource: "maps", withExtension: "xdc", subdirectory: "Assets") {
+                dcContext.setWebxdcIntegration(filepath: mapsXdc.path)
+                webxdcInstanceId = dcContext.initWebxdcIntegration(for: chatId)
+            }
+        }
+        super.init(dcContext: dcContext, messageId: webxdcInstanceId)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        navigationItem.rightBarButtonItem = nil
```

Comment:
I wondered why this is necessary, but the `WebxdcViewController` is setting a `More`-button and this is to override it, right?

Reply:
yes. the webxdc menu with "source code" and esp. "share" does not fit and is not needed